### PR TITLE
{lib}[gompi/2019a,gompic/2019a,iimpic/2019a] Boost.Python v1.70.0 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/0705ca05339a6da30c53b67874ee6e1b for a full test report.

File: easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.70.0-gompic-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+sources = ['boost_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['Boost-%(version)s_fix-Python3.patch']
+checksums = [
+    '882b48708d211a5f48e60b0124cf5863c1534cd544ecd0664bb534a4b5d506e9',  # boost_1_70_0.tar.gz
+    '48d1379df61e8289c5f98ff024a2b0429d49a7e06d7d88186d44a0a0254c4347',  # Boost-1.70.0_fix-Python3.patch
+]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [('Boost', version)]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.70.0-gompic-2019a.eb]: doesn't this require a `Boost-1.70.0-gompic-2019a.eb` easyconfig?

Comment (boegel) [reply easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.70.0-gompic-2019a.eb:23]: Yes, and that's there (otherwise the tests would fail)

Comment (boegel) [reply easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.70.0-gompic-2019a.eb:23]: Hmm, seems like it's no longer there in `develop`...

Apparently it got (incorrectly) archived in #8557, so I'll restore it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2631.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3ad62ff7af6982f0e6a93d8be3a7c881 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3159.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4d5c33c908e037fa28483e8daf842dc6 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**